packages/table/src/table.tsx

```diff
@@ -1004,6 +1092,53 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         this.handleSelection([]);
     }
 
+    private handleFocusMove = (e: KeyboardEvent, direction: "up" | "down" | "left" | "right") => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const {focusedCellCoordinates} = this.state;
+        const newFocusedCellCoordinates = { col: focusedCellCoordinates.col, row: focusedCellCoordinates.row };
+        const { grid } = this;
+        if (direction === "up") {
+            newFocusedCellCoordinates.row--;
```

Comment:
nit: I've worked on codebases where `++` and `--` were considered an antipattern unless used in a `for` loop. I tend to prefer an explicit `+= 1` and `-= 1`.